content/home/examples/a-stateful-component.md

```diff
@@ -4,4 +4,4 @@ order: 1
 domid: timer-example
 ---
 
-Además de obtener datos de entrada (a los que accedes a través de `this.props`),un componente puede tener datos en su estado internamente (a los que accedes a través de `this.state`). Cuando el estado del componente cambie de datos, el resultado se actualiza volviendo a invocar a `render()`.
+Además de obtener datos de entrada (a los que accedes a través de `this.props`), un componente puede tener datos en su estado interno (a los que accedes a través de `this.state`). Cuando los datos del estado de un componente cambian, el cambio va a ser actualizado reinvocando `render()`.
```

Comment:
```Cuando los datos del estado de un componente cambian, el cambio va a ser actualizado reinvocando `render()````

Sounds pretty ambiguous, what if we add something between the lines of 

```
Cuando los datos del estado de un componente cambian, se vuelve a invocar `render` con los nuevos valores en `this.state`
```

I know this is not an exact word by word translation, but makes more sense than what we have now.

Reply:
It sounds better.